modules/ding_user/ding_user.install

```diff
@@ -30,6 +31,18 @@ function ding_user_create_entity_cache_table() {
 }
 
 /**
+ * Creates the ding_private_key variable.
+ */
+function ding_user_create_private_key() {
+  // variable may be set in local.settings.php, but not in db.
+  if ($key = variable_get('ding_private_key', 0)) {
```

Comment:
Naming here is off.

If this variable is owned by `ding_user` then it must be prefixed with `ding_user` e.g. `'ding_user_private_key'. Otherwise it must be managed by the `ding2` profile.